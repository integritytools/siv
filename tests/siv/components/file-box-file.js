const React = require('react');
const FileBoxFile = require('../../../app/siv/file-box-file');
const renderer = require('react-test-renderer');

describe('<FileBoxFile />', () => {
  it('Renders correctly: Active state', () => {
    const component = renderer.create(
      <FileBoxFile
        currentImg="/path/to/some/img.jpg"
        fileBoxID={0}
        filePath="/path/to/some/img.jpg"
        onClick={jest.fn()}
      />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
  it('Renders correctly: Inactive state', () => {
    const component = renderer.create(
      <FileBoxFile
        currentImg="/path/to/some/img.jpg"
        fileBoxID={0}
        filePath="/path/to/some/other/img.jpg"
        onClick={jest.fn()}
      />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
