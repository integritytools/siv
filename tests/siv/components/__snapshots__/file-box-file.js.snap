exports[`<FileBoxFile /> Renders correctly: Active state 1`] = `
<li>
  <a
    className="active"
    href=""
    onClick={[Function anonymous]}>
    img
  </a>
</li>
`;

exports[`<FileBoxFile /> Renders correctly: Inactive state 1`] = `
<li>
  <a
    className=""
    href=""
    onClick={[Function anonymous]}>
    img
  </a>
</li>
`;
